jena-project/pom.xml

```diff
@@ -1,867 +1,828 @@
-            </execution>
-          </executions>
-          <configuration>
-            <rules>
-              <requirePluginVersions>
-                <message>Best practice is to always define plugin versions!</message>
-                <banLatest>true</banLatest>
-                <banRelease>true</banRelease>
-              </requirePluginVersions>
-              <requireReleaseDeps>
-                <message>No SNAPSHOT dependencies are allowed!</message>
-                <onlyWhenRelease>true</onlyWhenRelease>
-              </requireReleaseDeps>
-              <DependencyConvergence />
-            </rules>
-            <fail>true</fail>
-          </configuration>
-        </plugin>
-
-        <plugin>
-          <groupId>org.apache.felix</groupId>
-          <artifactId>maven-bundle-plugin</artifactId>
-          <version>2.5.3</version>
-          <extensions>true</extensions>
-        </plugin>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-shade-plugin</artifactId>
-          <version>2.4.3</version>
-        </plugin>
-
-
-      </plugins>
-    </pluginManagement>
-  </build>
-
-  <reporting>
-    <!-- <outputDirectory>${project.build.directory}/site</outputDirectory> -->
-
-    <plugins>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-report-plugin</artifactId>
-        <version>2.18.1</version>
-        <configuration>
-          <outputDirectory>${project.basedir}/target/surefire-reports-html</outputDirectory>
-        </configuration>
-      </plugin>
-
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>cobertura-maven-plugin</artifactId>
-        <version>2.7</version>
-        <configuration>
-          <instrumentation>
-            <ignores>
-              <ignore>org.slf4j.*</ignore>
-            </ignores>
-          </instrumentation>
-        </configuration>
-      </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-checkstyle-plugin</artifactId>
-        <version>2.15</version>
-      </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-jxr-plugin</artifactId>
-        <version>2.5</version>
-      </plugin>
-
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-pmd-plugin</artifactId>
-        <version>3.4</version>
-        <configuration>
-          <linkXref>true</linkXref>
-          <sourceEncoding>utf-8</sourceEncoding>
-          <minimumTokens>100</minimumTokens>
-        </configuration>
-      </plugin>
-
-      <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>findbugs-maven-plugin</artifactId>
-        <version>3.0.1</version>
-      </plugin>
-
-    </plugins>
-
-  </reporting>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
```

Comment:
This change is reported as: +831 −869 because tabs have been used.

1. Can't see what has materially changed.
1. It changes the licences header
1. Please - no tabs - display can be bad on the web (worse it if it gets mixed)
1. It asks in CONTRIBUTING.md :grin:

Reply:
Wait, something has gone wrong here. The only changes I made were to the values of a few properties in that `pom.xml`. I have no idea why it's doing such a giant diff. I need to figure that out.